src/howler.js

```diff
@@ -0,0 +1,7 @@
+var Howler
+
+if (typeof window !== 'undefined') {
+  Howler = require('howler')
+}
+
+module.exports = Howler
```

Comment:
This file would not be needed if Howler does error handling for the navigator error on server side. But normally howler is not requested on server side I suppose.